README.md

```diff
@@ -93,6 +93,24 @@ Run mock server: `lein run -m re-crud.mock-http-server`
 
 `make test` (requires phantomjs)
 
+## Dev Workflow
+
+- Change working directory to `example-app`:
+```
+$ cd path/to/re-crud/example-app
+```
+- Start REPL:
+```
+$ rm -rf src/cljs/re_crud
```

Comment:
This is not required the first time, may be add a note to mention why it is needed?